package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ui-kit",
```

Comment:
**Pros**
- Easy to setup.
- Works OOTB (except for result templates).
- Autocomplete for components, html attributes.
- Displays an error when a required attribute is missing.

**Cons**
- Initialization is non-idiomatic. Hiding it inside a wrapper gives an invalid hook error (to investigate).
- Custom result templates do not work (condition is using a custom `MapProp` decorator, which the stencil react plugin does not recognize)
- Component options are lost amid a sea of html attributes rather than being shown as props.
- Html attributes are not forwarded (e.g. specifying `placeholder` on `AtomicSearchBox` does not do anything). The autocomplete is therefore misleading.
- No access to the `controller` instance to render a different output.

**Overall impressions**
Working with Atomic components inside a ts instead of a html is a nicer dev experience. However, the components feel like they are not native React components. Certain functionality either does not work, or needs to be done in a non-idiomatic way.